toplevel/coqloop.ml

```diff
@@ -292,6 +278,9 @@ let coqloop_feed (fb : Feedback.feedback) = let open Feedback in
   | FileDependency (_,_) -> ()
   | FileLoaded (_,_) -> ()
   | Custom (_,_,_) -> ()
+  (* Re-enable when we switch back to feedback-based error printing *)
```

Comment:
What I don't get is why you need to ignore that feedback.
`coqc foo` should never generate it.